### PR TITLE
Create Pangram_in_python.py

### PR DESCRIPTION
**Pull request by sunilgknair051**
A neat and simple python3  for Pangram checks, replace the string to check and variations in ASCII characters would work as well.

## Have you read the Contribution Guidelines?

Yes.

## Description

A pangram is a string that contains all the letters of the English alphabet.
An example of a pangram is "The quick brown fox jumps over the lazy dog".
I have tried adding a variation in Python3.

## Checklist
- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue #596 
<br/>

### CODE REVIEW

File: Python/Pangram_in_python.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      
+# Sample code 
+string = 'the quick brown fox jumps over the lazy dog'
+if(ispangram(string) == True): 
+    print("Given String is a Pangram") 
+else: 
+    print("Given String is not Pangram") 
+    
+    
+/*
```
Comment (sukritishah15) [inline Python/Pangram_in_python.py:23]: Make `/*` to `'''`
It will then be a multiline comment in python.
Similarly `*/` to `'''`

**Review by sukritishah15 — CHANGES_REQUESTED** [review]
Once, you make the changes in this PR. I'll merge it along with #674 
Else, you can add the updated README.md file here in this PR and close #674 and I'll merge this one, only.
@sunilgknair051

File: Python/Pangram_in_python.py (hunk 1 above)
Comment (sunilgknair051) [reply Python/Pangram_in_python.py]: done!

**Comment by sunilgknair051** [comment]
> Once, you make the changes in this PR. I'll merge it along with #674
> Else, you can add the updated README.md file here in this PR and close #674 and I'll merge this one, only.
> @sunilgknair051

Closed the other PR, you can check this currently.

**Pull request merged**